tests/Model/Generated/Object/NotificationUrlTest.php

```diff
@@ -0,0 +1,282 @@
+<?php
+
+namespace bunq\test\Model\Object;
+
+use bunq\Exception\BunqException;
+use bunq\Model\Core\BunqModel;
+use bunq\Model\Generated\Endpoint\BunqMeTab;
+use bunq\Model\Generated\Endpoint\ChatMessageAnnouncement;
+use bunq\Model\Generated\Endpoint\DraftPayment;
+use bunq\Model\Generated\Endpoint\MasterCardAction;
+use bunq\Model\Generated\Endpoint\MonetaryAccountBank;
+use bunq\Model\Generated\Endpoint\Payment;
+use bunq\Model\Generated\Endpoint\PaymentBatch;
+use bunq\Model\Generated\Endpoint\RequestInquiry;
+use bunq\Model\Generated\Endpoint\RequestResponse;
+use bunq\Model\Generated\Endpoint\ScheduleInstance;
+use bunq\Model\Generated\Endpoint\SchedulePayment;
+use bunq\Model\Generated\Endpoint\ShareInviteBankInquiry;
+use bunq\Model\Generated\Endpoint\ShareInviteBankResponse;
+use bunq\Model\Generated\Object\NotificationUrl;
+use bunq\test\BunqSdkTestBase;
+use bunq\Util\FileUtil;
+
+/**
+ * Tests:
+ *  NotificationUrl
+ */
+class NotificationUrlTest extends BunqSdkTestBase
+{
+    /**
+     * Getter string constants
+     */
+    const GET_PAYMENT = 'getPayment';
+    const GET_BUNQ_ME_TAB = 'getBunqMeTab';
+    const GET_CHAT_MESSAGE_ANNOUNCEMENT = 'getChatMessageAnnouncement';
+    const GET_DRAFT_PAYMENT = 'getDraftPayment';
+    const GET_MASTER_CARD_ACTION = 'getMasterCardAction';
+    const GET_MONETARY_ACCOUNT_BANK = 'getMonetaryAccountBank';
+    const GET_PAYMENT_BATCH = 'getPaymentBatch';
+    const GET_REQUEST_INQUIRY = 'getRequestInquiry';
+    const GET_REQUEST_RESPONSE = 'getRequestResponse';
+    const GET_SCHEDULE_PAYMENT = 'getScheduledPayment';
+    const GET_SCHEDULE_INSTANCE = 'getScheduledInstance';
+    const GET_SHARE_INVITE_BANK_INQUIRY = 'getShareInviteBankInquiry';
+    const GET_SHARE_INVITE_BANK_RESPONSE = 'getShareInviteBankResponse';
+
+    /**
+     * Assertion errors.
+     */
+    const ASSERT_SHOULD_NOT_REACH_THIS_CODE_ERROR = 'Something super weird just happen';
+    const ASSERT_JSON_DECODE_ERROR = 'Might be that the JSON file is not a valid json.';
+    const ASSERT_OBJECT_IS_NULL_ERROR = 'Object seems to be null.';
+
+
+    /**
+     * Model json paths constants.
+     */
+    const BASE_PATH_JSON_MODEL =  __DIR__ . '/../../../resource/NotificationUrlJsons';
```

Comment:
From old code: why do we call the resource directory `resource` and not `Resource`? The rest seem to be capitalised :)